src/main/java/jeresources/util/RenderHelper.java

```diff
@@ -79,7 +79,7 @@ public static void renderEntity(MatrixStack matrixStack, int x, int y, double sc
         float rotationPitch = livingEntity.rotationPitch;
         float prevRotationYawHead = livingEntity.prevRotationYawHead;
         float rotationYawHead = livingEntity.rotationYawHead;
-        net.minecraft.client.renderer.RenderHelper.setupGuiFlatDiffuseLighting();
+        //net.minecraft.client.renderer.RenderHelper.setupGuiFlatDiffuseLighting();
```

Comment:
Feel free to remove the line if it was not needed

Reply:
Thing is, I think this line is needed for the mob rendering, but since that is disabled the line is causing said bug (for now). That's the reason why I only commented it out. I can of course remove it if you want it, you just have to remember the line if you re-enable the mob rendering.

Reply:
Leave commented out, but add a note that it is needed for mob rendering and to be re-added later